R/cBioPortal.R

```diff
@@ -100,15 +100,24 @@ utils::globalVariables(c("clinicalAttributeId", "value", "sampleId"))
 #' getGenePanel(api = cbio, genePanelId = "IMPACT341")
 #'
 #' @export
-cBioPortal <- function() {
+cBioPortal <- function(apiHostUrl) {
+    if (missing(apiHostUrl)) {
+        apiHost <- "www.cbioportal.org"
+        apiUrl <- "https://www.cbioportal.org/api/api-docs"
+    } else {
+        library(stringr)
+        apiHost <- str_split(apiHostUrl,"//")[[1]][2]
```

Comment:
are you talking about changing the subdomain from `www` to `beta`?

Reply:
@LiNk-NY Thanks for the quick review! It's not so much subdomain, but rather if people run a local instance of cBioPortal. So e.g. the API docs might be at `http://localhost:808/api/api-docs`  or `http://myinstitution.org/api/api-docs`